lib/module/plugin.js

```diff
@@ -20,7 +20,7 @@ export default function (ctx, inject) {
   <%=
   options.strategies.map(strategy => {
     const scheme = 'scheme_' + hash(options.strategyScheme.get(strategy))
-    const schemeOptions = JSON.stringify(strategy)
+    const schemeOptions = serialize(strategy)
```

Comment:
Serializing functions is often unstable and limited. PS this causes strings to be escaped